common/src/main/resources/messages/messages.properties

```diff
@@ -332,8 +332,9 @@ stack.sync.versions.from.cm.to.db.failed=Reading CM and active parcel versions f
 stack.sync.instance.updated=Updated metadata of instance {0} to {1} as the cloud provider reported it as {1}.
 stack.sync.instance.failed=Updated metadata of instance {0} to failed, because update was in progress, but instance isn''t member of the cluster.
 
-datalake.upgrade=Upgrade process initiated, target image id is {0}.
-datalake.upgrade.preparation=Upgrade preparation process initiated, target image id is {0}.
+datalake.upgrade=Upgrade process initiated for CDH {0}, the target image id is: {1}.
+datalake.rolling.upgrade=Rolling upgrade process initiated for CDH {0}, the target image id is: {1}.
+datalake.upgrade.preparation=Upgrade preparation process initiated for CDH {0}, target image id is {1}.
```

Comment:
I'm not sure we should refer to it as CDH, maybe just call it runtime

Reply:
you have fixed 1 line out of the 3

Reply:
Sorry, I missed those. It's fixed now.